assets/pages/oneoff-contributions/components/paymentMethodsContainer.jsx

```diff
@@ -15,6 +15,7 @@ function mapStateToProps(state) {
     email: state.user.email,
     error: state.oneoffContrib.error,
     hide: state.user.email === '' || state.user.fullName === '',
+    paymentStatus: 'NotStarted',
```

Comment:
Is this because the field requires a value, even though we're not using this logic for one-off?

Reply:
Yes.  I think that is okay?

Reply:
Yeah it's fine, just double-checking.

In general I'm thinking that we may not want to share the `paymentMethods` component between the two flows any more. There's now a fair bit of difference between them. Will look at this as part of the payment refactor card.